molo/yourwords/templates/yourwords/thank_you.html

```diff
@@ -13,10 +13,12 @@
   </div>
 
 <div class="block {{competition.thank_you_page.get_parent_page.get_effective_extra_style_hints}}">
+  {% load_thank_you_page_for_competition competition as thank_you_pages %}
+  {% for page in thank_you_pages %}
     {% if competition.thank_you_page %}
```

Comment:
This if statement now needs to change. It needs to be against `thank_you_pages` and it needs to be outside the for loop. So that if there are no pages, it should show the default message